examples/listlibs.go

```diff
@@ -54,7 +54,7 @@ func main() {
 
 }
 
-func findProcWithLib(r *regexp.Regexp, ps []process.Process) (matches map[process.Process][]string, harderror error, softerrors []error) {
+func findProcWithLib(r *regexp.Regexp, ps []process.Process) (matches map[process.Process][]string, softerrors []error, harderror error) {
```

Comment:
Does this even compile? You are changing the order of the parameters, but at the end we are returning the things in the wrong order..